WordPress/src/main/java/org/wordpress/android/ui/pages/PagesFragment.kt

```diff
@@ -400,6 +416,44 @@ class PagesFragment : Fragment(), GutenbergWarningDialogClickInterface {
         )
         dialog.show(fragmentManager, page.id.toString())
     }
+
+    override fun onStart() {
+        super.onStart()
+        EventBus.getDefault().registerSticky(this)
+    }
+
+    override fun onStop() {
+        super.onStop()
+        EventBus.getDefault().unregister(this)
+    }
+
+    @Subscribe(sticky = true, threadMode = ThreadMode.MAIN)
+    fun onEvent(event: QuickStartEvent) {
+        if (!isAdded || view == null) {
+            return
+        }
+
+        EventBus.getDefault().removeStickyEvent(event)
+        quickStartEvent = event
+
+        if (quickStartEvent?.task == QuickStartTask.CREATE_NEW_PAGE) {
+            view?.post {
+                val title = QuickStartUtils.stylizeQuickStartPrompt(
+                        requireActivity(),
+                        R.string.quick_start_dialog_create_new_page_message_short_pages,
+                        R.drawable.ic_create_white_24dp
```

Comment:
The `ic_create_white_24dp` icon in the floating action button changes from `white` to `blue_light` when the snackbar is shown.  The icon remains `blue_light` even after the snackbar has been hidden as well as leaving and returning to ***Site Pages***.  I'm not sure what's happening, but changing the `ic_create_white_24dp` icon to another icon (e.g. `ic_globe_white_24dp`) resolves the issue so it must be related to both the floating action button and the snackbar using the same resource.

Reply:
This is pretty interesting! Looks like drawables share a state, and tinting it in the snackbar affects all the current instances of drawable. I fixed it in [b68ec2e](https://github.com/wordpress-mobile/WordPress-Android/pull/8746/commits/b68ec2e726fc3b8f340ea4ef0ee6fc5e4993e480)